### PR TITLE
add troubleshoot in readme on import error

### PR DESCRIPTION
**Pull request by 0xaptosj**
### Description

We should document this import error due to tsconfig, im putting it in readme for now.

### CODE REVIEW

**Comment by 0xaptosj** [comment]
i don't have merge access, if it looks good please merge for me

**Review by xbtmatt — COMMENTED** [review]
Hm, I think we might want to be more general about how we approach troubleshooting. This is more like a single item in a troubleshooting faq

Maybe we should just link them to a full working example template

File: README.md [hunk 1]
```diff
@@ -131,6 +131,16 @@ To run the SDK tests, simply run from the root of this repository:
 pnpm test
 ```
 
+## Troubleshooting
+
+If you see import error when you do this
+
+```javascript
```
Comment (gregnazario) [inline README.md:138]: Nit I think this should be typescript since it only applies to typescript

**Review by gregnazario — APPROVED** [review]

**Comment by gregnazario** [comment]
I agree, we should probably make a docs page somewhere for troubleshooting.

We have a few flushed out examples with TypeScript already, which a user could copy paste.  We'd likely just want to focus on the onboarding experience for create aptos dapp

File: README.md (hunk 1 above)
Comment (0xaptosj) [reply README.md]: fixed

**Review by xbtmatt — APPROVED** [review]

**Pull request merged**